README.md

```diff
@@ -20,15 +20,15 @@ Vue based user interface for [elemental-chat DNA](https://github.com/holochain/e
    1. In one terminal window install and run the dna with with `hc` like this:
 
    ```shell
-   hc generate -a elemental-chat-test ../elemental-chat/elemental-chat.dna.gz
-   hc run -l -p 8888
+   hc sandbox gen ./tests/integration/setup/bundle/elemental-chat.happ -a="elemental-chat-1"
```

Comment:
We should have them download the `.happ` file and not have to push up a new `.happ` file to this repo everytime there is a change.

Reply:
This makes sense bug we can't do it until we can set the `skip_proof` in `hc sandbox get ...`